lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -148,6 +154,11 @@ $(document).ready(function() {
     }
   }
 
+  if (match = new RegExp(/\?redirected\_from=([^?]*)/).exec(window.location.href)) {
```

Comment:
Surprising that there isn't a built-in jQuery function for getting query string arguments, but fine as long as this works!

It looks like the regex will only match if `redirected_fom` is the first parameter after the `?` though. Is that too strict?

Reply:
Well, there is [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) but that is not supported by IE. This regex should work cross-browser.

> It looks like the regex will only match if redirected_fom is the first parameter after the ? though.

Would this be better? `[&?]redirected\_from=([^?]*)`

Reply:
I've made the regex more lenient.